src/main/java/com/example/atm/controller/AtmController.java

```diff
@@ -33,11 +44,32 @@ public AtmController(AtmService atmService, AccountService accountService,
         this.personService = personService;
     }
 
+    @GetMapping("/get-atms")
```

Comment:
```suggestion
    @GetMapping("/")
```
По RESTful best practices - для отримання списку всіх достатньо звернутися на таку url

Reply:
+